os/board/rtl8721csm/src/component/os/tizenrt/tizenrt_service.c

```diff
@@ -681,30 +704,87 @@ static int _tizenrt_create_task(struct task_struct *ptask, const char *name, u32
 	ptask->task = NULL;
 	ptask->task_name = NULL;
 	return _FAIL;
+#else
+	int func_addr, ctx_addr;
+	pid_t pid;
+	char str_func_addr[8+1];
+	char str_ctx_addr[8+1];
+	char *task_info[2 + 1];
+	priority = SCHED_PRIORITY_DEFAULT + priority;
+	priority = (priority > SCHED_PRIORITY_MAX || priority < SCHED_PRIORITY_MIN)?SCHED_PRIORITY_DEFAULT:priority;
+	stack_size *= sizeof(uint32_t);
+	/* Execute loading thread for load all binaries */
+	func_addr = (int)func;
+	ctx_addr = (int)thctx;
+	task_info[0] = itoa(func_addr, str_func_addr, 16);
```

Comment:
could you use any meaningful variable for indexing?
e.g. task_info[index].

Reply:
Is that necessary? By refer from os/kernel/binary_manager/binary_manager_recovery.c Line:349, it used the same indexing.